api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -253,13 +253,11 @@ public Order getOrderByOrderNumber(String orderNumber) {
 	 */
 	@Override
 	public Long getNextOrderNumberSeedSequenceValue() {
-		Criteria searchCriteria = sessionFactory.getCurrentSession().createCriteria(GlobalProperty.class);
```

Comment:
Am struggling to marry these two line deletions with the pull request message. :)